### PR TITLE
[BEAM-471][BEAM-472] Fixes to RAT configuration

### PR DESCRIPTION
**Pull request by kennknowles**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by kennknowles** [comment]
R: @dhalperi OR @jbonofre

File: pom.xml [hunk 1]
```diff
@@ -755,6 +737,36 @@
...
+              <exclude>.github/**/*</exclude>
+              <exclude>**/*.iml</exclude>
+              <exclude>**/package-list</exclude>
+              <exclude>**/user.avsc</exclude>
+              <exclude>**/test/resources/**/*.txt</exclude>
+              <exclude>**/test/**/.placeholder</exclude>
+              <exclude>.repository/**/*</exclude>
+
+              <!-- Default eclipse excludes neglect subprojects -->
+              <exclude>**/.checkstyle</exclude>
```
Comment (dhalperi) [inline pom.xml]: move `.iml` down here and generalize to `IDE`?

**Comment by dhalperi** [comment]
LGTM

**Comment by dhalperi** [comment]
Please self-merge once green.

**Pull request merged**